service/history/execution/mutable_state_task_generator.go

```diff
@@ -554,3 +568,18 @@ func (r *mutableStateTaskGeneratorImpl) getTargetDomainID(
 
 	return targetDomainID, nil
 }
+
+func getNextDecisionTimeout(attempt int64, defaultStartToCloseTimeout time.Duration) time.Duration {
+	if attempt <= 1 {
+		return defaultStartToCloseTimeout
+	}
+
+	nextInterval := float64(defaultInitIntervalForDecisionRetry) * math.Pow(2, float64(attempt-2))
+	nextInterval = math.Min(nextInterval, float64(defaultMaxIntervalForDecisionRetry))
+	jitterPortion := int(0.2 * nextInterval)
```

Comment:
nit: define the jitter coefficient as a const at the beginning of the file and use `(1 - coefficient)` instead of 0.8 below?

Reply:
addressed